lib/atom.cpp

```diff
@@ -83,7 +83,7 @@ double TransitionMatrix::totalRate() {
  * @param  dx:   Logarithmic step of the grid (default = 0.005)
  * @retval
  */
-Atom::Atom(int Z, double m, int A, NuclearRadiusModel radius_model, double fc,
+Atom::Atom(int Z, double m, int A, double radius, NuclearRadiusModel radius_model, double fc,
```

Comment:
documentation comment `@param  radius: ` missing for the new parameter
[reference](https://github.com/muon-spectroscopy-computational-project/mudirac/blob/1c3189358ca8928d251238adf9eb0678edb93975/lib/atom.cpp#L78)